components/transaction/UserGems.tsx

```diff
@@ -0,0 +1,62 @@
+import { StyleSheet, View, Image } from 'react-native';
+import Colors from '../../theme/colors';
+import { useGetTransactionQuery } from './api';
+import FetchError from '../common/feedback/FetchError';
+import CenteredFeedback from '../common/feedback/CenteredFeedback';
+import LText from '../common/Text';
+
+const UserGems = () => {
+  const { data, isError } = useGetTransactionQuery();
+
+  if (isError) {
+    return <FetchError />;
+  }
+
```

Comment:
Please add `isLoading` here so it won't display the error below if loading takes more time than we think.